city_scrapers/spiders/chi_northern_il_university.py

```diff
@@ -0,0 +1,111 @@
+                if href is None:
+                    continue
+                else:
+                    link = response.urljoin(href)
+                self.link_date_map.setdefault(new_date_obj, []).append({
+                    "title": "".join(name),
+                    "href": link
+                })
+        yield response.follow(
+            "https://www.neiu.edu/about/board-of-trustees/calendar-of-meetings",
+            callback=self._parse_detail
+        )
+
+    def _parse_detail(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        items_list = response.xpath(
+            '//div[@class="field field--name-field-generic-body field--type-text-long \
+field--label-hidden field--item"]'
+        )
+        year_list = items_list.xpath('.//h2')
+        for years in year_list:
+            year = re.search(r'\d{4}', years.extract()).group(0)
+            for items in years.xpath('.//following-sibling::ul[1]/li'):
+                item = items.extract()
+                start = self._parse_start(items, year)
+                meeting = Meeting(
+                    title=self._parse_title(items),
+                    description='',
+                    classification=self._parse_classification(item),
+                    start=start,
+                    end=None,
+                    all_day=False,
+                    time_notes="See agenda for meeting time",
+                    location=self._parse_location(item),
+                    links=self.link_date_map.get(start, []),
+                    source=response.url,
+                )
+                if "Rescheduled" in meeting["title"]:
+                    meeting["status"] = "Rescheduled"
+                    meeting["title"] = re.sub(r"Rescheduled", "", meeting["title"]).strip()
+                else:
+                    meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+                yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        title_obj = item.xpath(".//text()").extract()
+        title = re.sub(r'\(.+\)', '', "".join(title_obj))
+        title = title.split("-")[-1].strip(" ")
+        return title
+
+    def _parse_classification(self, title):
+        """Parse or generate classification from allowed options."""
+        if 'committee' in title.lower():
+            return COMMITTEE
+        if "meeting" in title.lower():
+            return BOARD
+        return NOT_CLASSIFIED
+
+    def _parse_start(self, item, year):
+        """Parse start datetime as a naive datetime object."""
+        item = item.xpath(".//text()").extract()[0].split("-")[0].strip()
+        date = re.sub(r',|\xa0|\*|\*\*', '', item).strip(" ")
+        date_obj = datetime.strptime(date, '%A %B %d')
```

Comment:
If you're defaulting to a specific time, this can be simplified like this (also without setting intermediate variables that are returned without being used otherwise):

```python
return datetime.strptime(date + " 13", "%A %B %d %H")
```